packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/VSCode/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {
+  Collapsible,
+  Text,
+  Button,
+  Stack,
+  Link,
+} from '@codesandbox/components';
+import { useUpgradeFromV1ToV2 } from 'app/hooks/useUpgradeFromV1ToV2';
+
+export const VSCode = () => {
+  const { perform, loading, canConvert } = useUpgradeFromV1ToV2('Docker menu');
+
+  return (
+    <Collapsible title="VS Code" defaultOpen>
+      <Stack direction="vertical" gap={6} padding={2}>
+        <Text as="p" variant="muted" margin={0}>
+          To use VS Code together with CodeSandbox, you need to convert your
+          Browser Sandbox into a Cloud Sandbox.
+        </Text>
+        <Text as="p" variant="muted" margin={0}>
+          Cloud Sandboxes are an improved coding experience that runs your code
+          in the cloud. They allow you to run Docker, code in new languages, add
+          servers, databases, and much more.
+        </Text>
+        <Text as="p" variant="muted" margin={0}>
+          Don&apos;t worry-this sandbox will not be deleted. After converting,
```

Comment:
```suggestion
          Don&apos;t worry&mdash;this sandbox will not be deleted. After converting,
```

Reply:
Why are we actually using a dash here?

Reply:
@tristandubbeld the copy is a double dash, so using the char reference is the easiest way to make sure it renders correctly.